### PR TITLE
Merge stabilization into master

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
test vsi please

**Comment by CyrusNajmabadi** [comment]
retest windows_eta_open_prtest please

**Comment by CyrusNajmabadi** [comment]
retest windows_debug_unit32_prtest please

**Comment by CyrusNajmabadi** [comment]
test vsi please

**Comment by CyrusNajmabadi** [comment]
retest windows_vsi_p3_open_prtest please

**Comment by CyrusNajmabadi** [comment]
tagging @dotnet/roslyn-infrastructure 

There are stabilization changes i'd like to work with in master.  So i went ahead and just did the merge myself (fixing merge conflicts i knew would occur in Completion).

**Comment by CyrusNajmabadi** [comment]
retest windows_vsi_p3_open_prtest  please

**Comment by CyrusNajmabadi** [comment]
@dotnet-bot retest windows_vsi_p3_open_prtest please
// Previous failure: http://dotnet-ci.cloudapp.net/job/Private/job/dotnet_roslyn-internal/job/master/job/windows_vsi_p3_open_prtest/107/
// Retest reason:

**Comment by CyrusNajmabadi** [comment]
@dotnet-bot retest windows_vsi_p3_open_prtest please
// Previous failure: http://dotnet-ci.cloudapp.net/job/Private/job/dotnet_roslyn-internal/job/master/job/windows_vsi_p3_open_prtest/108/
// Retest reason:

File: build/Targets/VSL.Versions.targets [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <SystemCollectionsImmutableVersion>1.2.0-rc3-24102-00</SystemCollectionsImmutableVersion>
     <MicrosoftDiaSymReaderVersion>1.0.8-rc3-60429-03</MicrosoftDiaSymReaderVersion>
     <MicrosoftDiaSymReaderNativeVersion>1.4.0-rc2</MicrosoftDiaSymReaderNativeVersion>
+    <MicrosoftDiaSymReaderPortablePdbVersion>1.0.0</MicrosoftDiaSymReaderPortablePdbVersion>
```
Comment (jinujoseph) [inline build/Targets/VSL.Versions.targets]: @CyrusNajmabadi 

@tmat had asked me not to add MicrosoftDiaSymReaderPortablePdbVersion

Comment (CyrusNajmabadi) [inline build/Targets/VSL.Versions.targets]: We can remove it if it isn't necessary.  Would you like me to do that?

Comment (jinujoseph) [inline build/Targets/VSL.Versions.targets]: No , lets wait for @tmat take a look at this and the project.json version changes and he can send another PR with necessary changes. This should unblock you for now.

Comment (CyrusNajmabadi) [inline build/Targets/VSL.Versions.targets]: Sounds good.  Thanks!
